scripts/update-typedoc-functions-path.cjs

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env node
```

Comment:
did you write this whole script today?

why .cjs rather than .mjs?

Reply:
Yes - I paired with an LLM to to help arrive at the solution more quickly. This sort of task seemed like the perfect candidate. 

And I instinctively went for .cjs as this is a build script. Will note for the future .mjs works and is preferred!

Reply:
I'm skeptical of the amount of code here to work around the CF bug. Did https://github.com/TypeStrong/typedoc/issues/2111#issuecomment-1741049144 not work?

https://github.com/TypeStrong/typedoc/issues/2111#issuecomment-1741835099 is more work but looks soild.

I will approve this because it fixes the immediate problem:
https://agoric-sdk.pages.dev/funcs/_agoric_orchestration.buildRootObject
https://pc-bump-typedoc.agoric-sdk.pages.dev/funcs/_agoric_orchestration.buildRootObject

but please file an issue to come back and remove this file. I'd suggest to just merge this now but it's going to need a rebase anyway so please include in a doc comment why this file is needed, what alternatives were researched and what are possible next steps

Reply:
> Did TypeStrong/typedoc#2111 (comment) not work

It does not since we have JS enabled. Part of the script reaches into a build-generated base64 string in a js file that contains navigation state to also update the urls there.

> TypeStrong/typedoc#2111 (comment) is more work but looks soild

I didn't try this one - the git links are dead and it seems like `PlayForm/Build` might do something different now - I don't see any mentions of a typedoc theme in the [README](https://github.com/PlayForm/Build).  This also involves an extra dependency.

Reply:
> but please file an issue to come back and remove this file. I'd suggest to just merge this now but it's going to need a rebase anyway so please include in a doc comment why this file is needed, what alternatives were researched and what are possible next steps

We can leave #9729 open for this - i've updated to refs. PTAL at [092acb7](https://github.com/Agoric/agoric-sdk/pull/9727/commits/092acb7e3d1eaff2d68b587121773a1c76f5984a) for an updated file description

Reply:
Excellent.